src/domain/app/BrowserApp.tsx

```diff
@@ -21,15 +22,17 @@ if (process.env.NODE_ENV !== 'production') {
 
 // Export for testing purpose
 export const appRoutes = (
-  <Switch>
-    <Route exact path="/callback" component={OidcCallback} />
-    <Redirect exact path="/" to="/fi/home" />
-    <Route path={`/${localeParam}/*`} component={App} />
-    <Route exact path={`/${localeParam}/callback`} component={OidcCallback} />
-    <Route
-      render={props => <Redirect to={`/fi${props.location.pathname}`} />}
-    />
-  </Switch>
+  <PageLayout>
```

Comment:
You should be very sure that the layout is the same for all pages before automatically applying the layout by default on all routes, usually it's better to use the layout on every route "manually".

Reply:
Yeah i've been consider about that but for now i feel like every page should have header and footer so that's why (even a full page spinner). Spamming the layout was kinda more customizable but not needed here. 

I will remove the wrap and do manually in case we need one in future.